plugins/3MFReader/WorkspaceDialog.qml

```diff
@@ -96,7 +96,7 @@ UM.Dialog
                         WorkspaceRow
                         {
                             leftLabelText: catalog.i18nc("@action:label", manager.isPrinterGroup ? "Printer Group" : "Printer Name")
-                            rightLabelText: manager.machineName
+                            rightLabelText: manager.machineName == "Create new" ? "" : manager.machineName
```

Comment:
Don't we want to translate the "Create new" string?

Reply:
I do think it's a bit cleaner if the machineName is fully set through the manager. So if no compatible machine is found the `ThreeMFWorkspaceReader.py` sets `machineName="Create new".